linux_os/guide/system/accounts/accounts-pam/pam_disable_automatic_configuration/oval/shared.xml

```diff
@@ -8,7 +8,8 @@
       <description>Verify the SUSE operating system is configured to not overwrite Pluggable
     Authentication Modules (PAM) configuration on package changes.</description>
     </metadata>
-    <criteria>
+    <criteria comment="package pam_apparmor is installed and automatic configuration is disabled" operator="AND">
+        <extend_definition comment="package package pam_apparmor installed" definition_ref="package_pam_apparmor_installed" />
```

Comment:
It is not necessary to include this test in this rule. The `platform: package[pam_apparmor]` included in `rule.yml` is enough.

Reply:
Hello @marcusburghardt,
I have implemented the proposed change.
All the best 
Rumen